src/Amazon.Lambda.Tools/TemplateProcessor/UpdatableResource.cs

```diff
@@ -179,6 +211,11 @@ public string GetMetadataDockerTag()
             {
                 return this.DataSource.GetValueDictionaryFromResource("Metadata", "DockerBuildArgs");
             }
+
+            private bool IsECRImage(string path)
```

Comment:
You can make this public and static and then add some unit tests for this method. Since this is a tool and not a library we can be more open to making things public and not worry about changing the API later.

Reply:
Done